File: num.java
public class EvenOddNumbers {
    public static void main(String[] args) {
        int N1 = 1;
        int N2 = 10;
        int sumEven = 0;
        int sumOdd = 0;

        System.out.println("Even numbers between " + N1 + " and " + N2 + ":");
        for (int i = N1; i <= N2; i++) {
            if (i % 2 == 0) {
                System.out.print(i + " ");
                sumEven += i;
            }
        }

        System.out.println("\nOdd numbers between " + N1 + " and " + N2 + ":");
        for (int i = N1; i <= N2; i++) {
            if (i % 2 != 0) {
                System.out.print(i + " ");
                sumOdd += i;
            }
        }

        System.out.println("\nSum of even numbers: " + sumEven);
        System.out.println("Sum of odd numbers: " + sumOdd);
    }
}

File: num.c
#include <stdio.h>

int main() {
    int N1, N2, i, sum_even = 0, sum_odd = 0;

    printf("Enter two numbers (N1 < N2): ");
    scanf("%d %d", &N1, &N2);

    printf("Even numbers between %d and %d are: ", N1, N2);
    for (i = N1; i <= N2; i++) {
        if (i % 2 == 0) {
            printf("%d ", i);
            sum_even += i;
        }
    }

    printf("\nOdd numbers between %d and %d are: ", N1, N2);
    for (i = N1; i <= N2; i++) {
        if (i % 2 != 0) {
            printf("%d ", i);
            sum_odd += i;
        }
    }

    printf("\nSum of even numbers: %d\n", sum_even);
    printf("Sum of odd numbers: %d\n", sum_odd);

    return 0;
}
